src/auth/auth-service.ts

```diff
@@ -0,0 +1,58 @@
+/* Facade for keycloak */
+import _ from 'lodash';
+import Keycloak from 'keycloak-js';
+import { getConfig } from '../config';
+import { loadTokens, removeTokens, storeTokens } from './token-store';
+
+let kc: Keycloak.KeycloakInstance;
```

Comment:
Import KeycloakInstance from "keycloak-js" and change type

import Keycloak, { KeycloakInstance } from 'keycloak-js';

Reply:
skal prøve, takk!